roles/sushy_emulator/tasks/collect_details.yml

```diff
@@ -79,13 +104,12 @@
   block:
     - name: Get Openstack instance UUIDs
       ansible.builtin.command:
-        cmd: "openstack --os-cloud={{ cifmw_sushy_emulator_driver_openstack_cloud }} server list --name {{ cifmw_sushy_emulator_instance_node_name_prefix }}.* -f json -c ID | jq -c [.[].ID])"
+        cmd: "openstack --os-cloud={{ cifmw_sushy_emulator_driver_openstack_cloud }} server list --name {{ cifmw_sushy_emulator_vm_prefix_filter }}.* -f json -c ID | jq -c [.[].ID])"
```

Comment:
We could use the openstack collection here?
```
openstack.cloud.server_info:
    cloud: "{{ cifmw_sushy_emulator_driver_openstack_cloud }}"
    name: "{{ cifmw_sushy_emulator_vm_prefix_filter }}*"
``

Reply:
We could but using the openstack collection releases dragons around what OpenStack clouds we can target, this will be running against a user provided OpenStack environment which may hit issues with [1].

Generally I like to use modules where I can but in this case I don't think it's worth it.

[1] https://opendev.org/openstack/ansible-collections-openstack#branches-and-non-backward-compatibility-

Reply:
mm, ok.